test/e2e/nutanix_test.go

```diff
@@ -775,3 +776,41 @@ func TestNutanixKubernetes126OIDC(t *testing.T) {
 	)
 	runOIDCFlow(test)
 }
+
+// Registry Mirror Tests
+func TestNutanixKubernetes124UbuntuRegistryMirrorAndCert(t *testing.T) {
```

Comment:
do you mind moving the e2e to a separate PR? We will need to setup some infra for this which we don't have currently.